executor/analyze.go

```diff
@@ -270,7 +270,7 @@ func (e *AnalyzeIndexExec) fetchAnalyzeResult(ranges []*ranger.Range, isNullRang
 	if err != nil {
 		return err
 	}
-	result.Fetch(ctx)
+	result.Fetch(ctx, tipb.EncodeType_TypeDefault)
```

Comment:
Would it be faster if we support chunk encoded data for the `Analyze` API?

Reply:
We can open an issue for this, leave it to the future optimizations.

Reply:
OK.